presto-main/src/main/java/io/prestosql/security/AccessControlModule.java

```diff
@@ -26,7 +31,24 @@
     public void configure(Binder binder)
     {
         binder.bind(AccessControlManager.class).in(Scopes.SINGLETON);
-        binder.bind(AccessControl.class).to(AccessControlManager.class).in(Scopes.SINGLETON);
         newExporter(binder).export(AccessControlManager.class).withGeneratedName();
     }
+
+    @Provides
+    @Singleton
+    public AccessControl createAccessControl(AccessControlManager accessControlManager)
+    {
+        Logger logger = Logger.get(AccessControl.class);
+
+        // TODO enable logging invocation when logging configuration was changed at the runtime
```

Comment:
what do we do with this?

Reply:
IMO it requires to implement `ForwardingAccessControl` it has about 20 methods to implement. I was too lazy for that today. Maybe it is great `good first issue` for somebody else.

Other than that I don't think many users know how to change logging configuration at runtime.

Reply:
> it has about 20 methods to implement.

That doesn't seem like a lot (just a bit of copy-paste). Should we address it in this PR too?

Reply:
It took me about 2h :)